cmd/client.go

```diff
@@ -55,10 +56,31 @@ func (cli *Client) RunNode(c *clipkg.Context) error {
 	defer app.Stop()
 	logNodeBalance(store)
 	logConfigVariables(config)
+	logIfNonceOutOfSync(store)
 
 	return cli.errorOut(cli.Runner.Run(app))
 }
 
+func logIfNonceOutOfSync(store *strpkg.Store) {
+	account := store.TxManager.GetActiveAccount()
+	lastNonce, err := store.GetLastNonce(account.Address)
+	if err == storm.ErrNotFound {
+		return
+	}
+
+	if localNonceIsNotCurrent(lastNonce, account.GetNonce()) {
+		logger.Warn("The account is being used by another wallet and is not safe to use with chainlink")
```

Comment:
In this scenario, do we want to update our local nonce? What do you think @se3000?

Reply:
I don't think we need to update the nonce. When creating a new transaction, we don't actually pull the nonce from the database. We get it from the store's active account, [where it's wrapped in a mutex](https://github.com/smartcontractkit/chainlink/blob/master/store/tx_manager.go#L246) to avoid race conditions. The active account currently gets the nonce from the blockchain, so updating the nonce in the database shouldn't be a problem. In the a subsequent PR, we should make the active account look at both the db and the blockchain and take the higher nonce, in which case updating the db should also not be a problem. The DB will ultimately be updated on the next transaction we create.